### ISSUE TITLE
[DDL] Complete Accessibility QA in Staging

### ISSUE THREAD

**Issue by jacobworrell** [department-of-veterans-affairs/va.gov-team#47904]
Complete Checklist in [TestRails](https://dsvavsp.testrail.io/index.php?/runs/view/3733&group_by=cases:section_id&group_order=asc)

**Comment by sortizsh** [comment]
@jerekshoe  please add estimate

**Comment by jacobworrell** [comment]
@jaredcunha please size this before you head off to retreat.

**Comment by jaredcunha** [comment]
These are some tests that we can safely remove. Most apply to handling video and audio files, which are not included in the designs. There is also a test we can remove that relates to multiple languages present on the page.:
- SC 1.2.1_A: Text alternative is provided for prerecorded audio or video files.
- SC 1.2.2_A Captions (Prerecorded)
- SC 1.2.4_A Captions (Live)
- SC 1.4.2_A Audio Control
- SC 3.1.2_AA Language of Parts

The rest of the rules should be criteria that we will be able to test.

**Comment by jaredcunha** [comment]
Everything that could be tested so far has been tested. The items that are still marked 'untested' are more yes/no than pass/fail, and I think we might be able to clean some of those items up so they are not even considered tests. @jacobworrell
